### ISSUE TITLE
Option `ProcessingOfPageHierarchy` = `HiearchyAsFolderTree` do not work

### ISSUE THREAD

**Issue by sky-blue** [alxnbl/onenote-md-exporter#63]
**Describe the bug**
I set `HiearchyAsFolderTree` to `ProcessingOfPageHierarchy`, but it was not in effect actually, no parentpage folder.

```json
  /* How page hierarchies should be handled :
     - HiearchyAsFolderTree : as parent folders of the page. Ex: section/parentpage/childpage.md
     - HiearchyAsPageTitlePrefix : as page filename prefix. Ex: section/paarentpage_childpage.md
     - IgnoreHierarchy : ignore page hierarchy. Ex: section/childpage.md
  */
  "ProcessingOfPageHierarchy": "HiearchyAsFolderTree",
```

**Desktop (please complete the following information):**
 - Windows version : Win10, 64 bits, 21H2, 19044.1586
 - Office Version : Office 365, 32 bits

**Comment by dreamflasher** [comment]
I can confirm this. Export as HiearchyAsFolderTree does not work.

**Comment by alxnbl** [comment]
Thank you for the report ! Solved in version 1.3.0.
